src/main/java/net/fabricmc/loader/impl/launch/knot/KnotClassLoader.java

```diff
@@ -61,6 +61,10 @@ KnotClassDelegate<?> getDelegate() {
 		return delegate;
 	}
 
+	public DynamicURLClassLoader getUrlLoader() {
```

Comment:
This one is unused and should be removed.

Reply:
This method provides developers with direct access to the URLClassLoader to access methods that have not been overridden.

Reply:
This method isnt part of the API, and exposes impl details (that the parent loader is of a DynamicURLClassLoader) that shouldnt be made into an API.

Reply:
> This method isnt part of the API, and exposes impl details (that the parent loader is of a DynamicURLClassLoader) that shouldnt be made into an API.

Okay, I'll remove it.